cli/command/swarm/opts.go

```diff
@@ -24,13 +24,17 @@ const (
 	flagToken               = "token"
 	flagTaskHistoryLimit    = "task-history-limit"
 	flagExternalCA          = "external-ca"
+	flagKeepOldSnapshots    = "keep-old-snapshots"
```

Comment:
`0` does not disable snapshots. It just avoids retaining extra snapshots. I don't think it's a good idea to support totally disabling snapshots because the log will grow without bound.